File: Ad-Hoc/__3000__/3405/3405.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        System.out.println(trailling_zeros(n));
        in.close();

    }

    public static int trailling_zeros(int n){
        int count = 0;
        while (n >= 5){
            n = (int) Math.floor((double) n / 5);
            count += n;
        }
        return count;
    }
}

File: Ad-Hoc/__3000__/3405/3405.cpp
#include <iostream>
#include <cmath>

using namespace std;

int trailing_zeros(int n){
    int count = 0;
    while (n >= 5){
        n = floor(n / 5);
        count += n;
    }
    return count;
}

int main(){
    int n;
    cin >> n;
    cout << trailing_zeros(n) << endl;

    return 0;
}